composer.json

```diff
@@ -70,7 +70,8 @@
         "zendframework/zendrest": "2.0.2",
         "zendframework/zendservice-amazon": "2.3.0",
         "zendframework/zendservice-recaptcha": "3.1.0",
-        "zf-commons/zfc-rbac": "2.6.3"
+        "zf-commons/zfc-rbac": "2.6.3",
+        "zendframework/zend-form": "^2.12"
```

Comment:
I would recommend pinning this to 2.12.0 for consistency with the way we handle other components. We've had problems in the past where even minor version changes have broken things, so I try to keep versions under tight control to prevent a `composer update` from having unanticipated side effects.